src/bun.js/test/expect.zig

```diff
@@ -113,6 +113,31 @@ pub const Expect = struct {
         return received ++ matcher_name ++ "<d>(<r>" ++ args ++ "<d>)<r>";
     }
 
+    pub fn isObjectEmpty(globalThis: *JSGlobalObject, value: JSValue) bool {
```

Comment:
This should be moved to `bindings.zig` as another method for `JSValue`. also jest-extended checks the length of `Object.keys` to determine if the object is empty. Can you add some more tests for non-enumerable properties and symbols?